controllers/os-ubuntu-alicloud/pkg/generator/templates/cloud-init.ubuntu-ali.template

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+{{- define "put-content" -}}
+cat << EOF | base64 -d > '{{ .Path }}'
+{{ .Content }}
+EOF
+{{- end -}}
+
+{{- if .Bootstrap }}
+#Disable upgrade related services
```

Comment:
the following `systemctl` commands are CoreOS specific, they are doing nothing on ubuntu - remove them.